evm/src/cpu/kernel/constants/global_metadata.rs

```diff
@@ -30,68 +30,64 @@ pub(crate) enum GlobalMetadata {
     TransactionTrieRootDigestAfter = 11,
     ReceiptTrieRootDigestAfter = 12,
 
-    /// The sizes of the `TrieEncodedChild` and `TrieEncodedChildLen` buffers. In other words, the
-    /// next available offset in these buffers.
-    TrieEncodedChildSize = 13,
-
     // Block metadata.
-    BlockBeneficiary = 14,
-    BlockTimestamp = 15,
-    BlockNumber = 16,
-    BlockDifficulty = 17,
-    BlockRandom = 18,
-    BlockGasLimit = 19,
-    BlockChainId = 20,
-    BlockBaseFee = 21,
-    BlockGasUsed = 22,
+    BlockBeneficiary = 13,
+    BlockTimestamp = 14,
```

Comment:
If you want to avoid the hassle of renaming, Rust wil auto increment the discriminant for you. See https://doc.rust-lang.org/reference/items/enumerations.html#implicit-discriminants

Reply:
Implemented in #1426 FYI.